.taskcluster.yml

```
@@ -147,6 +147,42 @@ tasks:
           owner: cdenizet@mozilla.com
           source: ${repository}/raw/${head_rev}/.taskcluster.yml
 
+      - $if: 'tasks_for == "github-push" && head_branch[:10] == "refs/tags/"'
+        then:
+          taskId: {$eval: as_slugid("test_mozilla_central")}
+          dependencies:
+            - {$eval: as_slugid("lint_test_task")}
+            - {$eval: as_slugid("windows_test_task")}
+          created: {$fromNow: ''}
+          deadline: {$fromNow: '1 hour'}
+          provisionerId: proj-relman
+          workerType: ci
+          payload:
+            maxRunTime: 3600
+            image: "rust:buster"
+            command:
+              - "/bin/bash"
+              - "-cx"
+              - "git clone --recursive --quiet ${repository} &&
+                 [ ! -d \"cache/gecko-dev\" ] &&
+                 git clone --quiet https://github.com/mozilla/gecko-dev.git /cache/gecko-dev || true &&
+                 pushd /cache/gecko-dev && git pull origin master && popd &&
+                 mkdir -p /tmp/mozilla_central_output &&
+                 cd rust-code-analysis &&
+                 git -c advice.detachedHead=false checkout ${head_rev} &&
+                 cargo build --release --all --all-features &&
+                 cargo run --release -p rust-code-analysis-cli -- -p /cache/gecko-dev \
+                                     -j24 --metrics -O json -o /tmp/mozilla_central_output"
```

```suggestion
                                     -j4 --metrics -O json -o /tmp/mozilla_central_output"
```

`j24` is probably too much, the `ci` machine is small (`n1-standard-4`), so let's use `j4` instead.